pages/vi/vi-github-and-repositories.md

```diff
@@ -26,6 +26,9 @@ This is just a summary of the steps that you will need to perform. Please, keep
 * [Configure a Remote Repository for Your Fork](#Configure_a_Remote_Repository_for_Your_Fork)
 * [Sync Your Fork](#Sync_Your_Fork)
 
+
+** Note :** You will see some common names used in git (which you can change), such as `master: the name of the main branch`, `upstream: the location where you forked the repository from`, and `origin: your github`. Both `upstream and origin` are considered remote. Keep in mind also, a repository may contain many branches.
```

Comment:
instead of `origin: your github` make it location you cloned repository from
 make `(which you can change)` italic and `remote`  bold